controllers/v2/unavailability/api.py

```diff
@@ -50,7 +55,7 @@ class VolunteerUnavailabilityV2(Resource):
 
     @requires_auth
     @marshal_with(volunteer_unavailability_time)
-    @is_user_or_has_role(None, UserType.ROOT_ADMIN)
+    # @is_user_or_has_role(None, UserType.ROOT_ADMIN)
```

Comment:
This should be secure, revert this change

Reply:
huh thats weird I swear I reverted that already will do